vespa-http-client/src/main/java/com/yahoo/vespa/http/client/SyncFeedClient.java

```diff
@@ -62,14 +62,19 @@ public void close() {
         private final String operationId;
 
         public SyncOperation(String documentId, CharSequence documentData) {
-            this(documentId, documentData, null);
+            this(documentId, documentData, null, null);
         }
 
-        public SyncOperation(String documentId, CharSequence documentData, Object context) {
+        public SyncOperation(String documentId, CharSequence documentData, String operationId) {
```

Comment:
This breaks the API by removing the existing 3-arg constructor. Please keep the current one (without operationId) and add a new taking operationId). The old one can call the new, with the operationId generated by the random generator.